src/main/java/com/uber/cadence/internal/worker/SingleWorkerOptions.java

```diff
@@ -43,6 +43,8 @@
 
     private Scope metricsScope;
 
+    private boolean enableLoggingInReplay = false;
```

Comment:
No need to specify value is it matches the default in Java. It is actually antipattern.

Reply:
Removed